common/model/db/views/dataNodeDef/view.js

```diff
@@ -44,12 +44,14 @@ export default class ViewDataNodeDef extends TableDataNodeDef {
     })
   }
 
-  get columnUuid() {
-    return this.getColumn(`${NodeDef.getName(this.nodeDef)}_uuid`)
+  get columnNodeDefUuid() {
+    // node def can be the entity itself or it's source entity (if virtual)
+    const { nodeDef } = this.tableData
+    return new ColumnNodeDef(this, nodeDef)
   }
 
-  get columnNodeDefUuid() {
-    return new ColumnNodeDef(this, this.nodeDef)
+  get columnUuid() {
+    return this.columnNodeDefUuid.namesFull[0]
```

Comment:
I would add a method to column nameFull. what do u think?

Reply:
ok